training/general/src/environments/lightgbm-3.2-ubuntu18.04-py37-cpu/asset.yaml

```diff
@@ -0,0 +1,10 @@
+name: lightgbm-3.2-ubuntu18.04-py37-cpu
```

Comment:
probably worth a better name. See naming convention for environments

Reply:
I'd separate lightgbm environment into a different PR

Reply:
changed name to lightgbm-3.3